src/VisualStudio/TestUtilities2/MockServiceProvider.vb

```diff
@@ -19,13 +19,15 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests
     Friend Class MockServiceProvider
         Implements IServiceProvider
         Implements SVsServiceProvider ' The shell service provider actually implements this too for people using that type directly
+        Implements SVsRunningDocumentTable
```

Comment:
I don't imagine you meant to write this -- nothing should be assuming the RDT directly implements the service marker interface.

Reply:
Yes that was a mistake.

Reply:
I've removed it.